cumulusci.yml

```diff
@@ -104,6 +115,14 @@ flows:
                     - when: "'npsp' not in tasks.get_installed_packages()"
                       action: hide
                 when: "org_config.has_minimum_package_version('npsp', '1.0')"
+            4:
+                flow: install_ofm_template
```

Comment:
@jstvz @sella24 Using this flow instead to optionally install OutboundFundsCommunity while also including the deploy_pre and deploy_post steps. Do either of you see any issues with taking this approach?

Reply:
No issues with this approach.

Reply:
@jstvz Sounds good, do we need to spin up a new staging installer in order to reflect these new changes? If so, you can spin that up for us?

Reply:
@seanpat09 https://metadeploy-stg.herokuapp.com/products/outbound-funds/65c7f649/install